### PR TITLE
Support for Prettier

### PR DESCRIPTION
**Pull request by Zyrakia**
This PR adds support for stylization of the output files, utilizing the Prettier API.

I made it default `false` since most people do not care about the stylization of any auto-generated code, but I personally enjoy cleaner output when debugging or checking the DTO for a quick reference.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,6 +71,7 @@ All parameters are optional.
 - [`classValidation`]: (default: `"false"`) - Add validation decorators from `class-validator`. Not compatible with `noDependencies = "true"` and `outputType = "interface"`.
 - [`noDependencies`]: (default: `"false"`) - Any imports and decorators that are specific to NestJS and Prisma are omitted, such that there are no references to external dependencies. This is useful if you want to generate appropriate DTOs for the frontend.
 - [`outputType`]: (default: `"class"`) - Output the DTOs as `class` or as `interface`. `interface` should only be used to generate DTOs for the frontend.
+- [`prettier`]: (default: `"false"`) - Should the output files should be stylized with Prettier?
```
Comment (Brakebein) [inline README.md]: Twice `shoud` in the sentence.
Or maybe just a simple phrase without question: `Stylize output files with prettier.`

File: README.md [hunk 2]
```diff
@@ -220,16 +222,16 @@ outputToNestJsResourceStructure = "true"
...
+id String @id @default(dbgenerated("gen_random_uuid()")) @db.Uuid
+/// @DtoReadOnly
+createdAt DateTime @default(now())
+/// @DtoRelationRequired
+createdBy User? @relation("CreatedQuestions", fields: [createdById], references: [id])
+createdById String? @db.Uuid
+updatedAt DateTime @updatedAt
+/// @DtoRelationRequired
+updatedBy User? @relation("UpdatedQuestions", fields: [updatedById], references: [id])
+updatedById String? @db.Uuid
```
Comment (Brakebein) [inline README.md]: Does your IDE auto-format the README file? At this point, it would be nice, if you revert the indentation.

**Review by Brakebein — CHANGES_REQUESTED** [review]
Thanks for the PR! Looks good to me. Just some minor changes in the readme.

File: README.md (hunk 2 above)
Comment (Zyrakia) [reply README.md]: Yes absolutely, I totally forgot. I'll do it that as soon as I'm home today.

**Comment by Zyrakia** [comment]
There we go, changes in this PR now actually reflect the meaning of this PR!

**Review by Brakebein — APPROVED** [review]

**Pull request merged**